server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/ProjectKeyPage.java

```diff
@@ -17,87 +17,84 @@
-    $("#project-key").should(exist);
+    Selenide.$("#project-key").should(Condition.exist);
   }
 
   public ProjectKeyPage assertSimpleUpdate() {
-    $("#update-key-new-key").shouldBe(visible);
-    $("#update-key-submit").shouldBe(visible);
+    Selenide.$("#update-key-new-key").shouldBe(Condition.visible);
+    Selenide.$("#update-key-submit").shouldBe(Condition.visible);
     return this;
   }
 
   public ProjectKeyPage trySimpleUpdate(String newKey) {
-    $("#update-key-new-key").val(newKey);
-    $("#update-key-submit").click();
-    $("#update-key-confirm").click();
+    Selenide.$("#update-key-new-key").val(newKey);
+    Selenide.$("#update-key-submit").click();
+    Selenide.$("#update-key-confirm").click();
     return this;
   }
 
   public ProjectKeyPage openFineGrainedUpdate() {
-    $("#update-key-tab-fine").click();
-    $("#project-key-fine-grained-update").shouldBe(visible);
+    Selenide.$("#update-key-tab-fine").click();
+    Selenide.$("#project-key-fine-grained-update").shouldBe(Condition.visible);
     return this;
   }
 
   public ProjectKeyPage tryFineGrainedUpdate(String key, String newKey) {
-    SelenideElement form = $(".js-fine-grained-update[data-key=\"" + key + "\"]");
-    form.shouldBe(visible);
+    SelenideElement form = Selenide.$(".js-fine-grained-update[data-key=\"" + key + "\"]");
+    form.shouldBe(Condition.visible);
 
     form.$("input").val(newKey);
     form.$("button").click();
 
-    $("#update-key-confirm").click();
+    Selenide.$("#update-key-confirm").click();
     return this;
   }
 
   public ProjectKeyPage assertBulkChange() {
-    $("#bulk-update-replace").shouldBe(visible);
-    $("#bulk-update-by").shouldBe(visible);
-    $("#bulk-update-see-results").shouldBe(visible);
+    Selenide.$("#bulk-update-replace").shouldBe(Condition.visible);
+    Selenide.$("#bulk-update-by").shouldBe(Condition.visible);
+    Selenide.$("#bulk-update-see-results").shouldBe(Condition.visible);
     return this;
   }
 
   public ProjectKeyPage simulateBulkChange(String replace, String by) {
-    $("#bulk-update-replace").val(replace);
-    $("#bulk-update-by").val(by);
-    $("#bulk-update-see-results").click();
+    Selenide.$("#bulk-update-replace").val(replace);
+    Selenide.$("#bulk-update-by").val(by);
+    Selenide.$("#bulk-update-see-results").click();
 
-    $("#bulk-update-simulation").shouldBe(visible);
+    Selenide.$("#bulk-update-simulation").shouldBe(Condition.visible);
     return this;
   }
 
   public ProjectKeyPage assertBulkChangeSimulationResult(String oldKey, String newKey) {
-    SelenideElement row = $("#bulk-update-results").$("[data-key=\"" + oldKey + "\"]");
-    row.$(".js-old-key").should(hasText(oldKey));
-    row.$(".js-new-key").should(hasText(newKey));
+    SelenideElement row = Selenide.$("#bulk-update-results").$("[data-key=\"" + oldKey + "\"]");
+    row.$(".js-old-key").should(Condition.hasText(oldKey));
+    row.$(".js-new-key").should(Condition.hasText(newKey));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "hasText"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)